gnucash/gnome-utils/gnc-gnome-utils.c

```diff
@@ -223,7 +223,7 @@ gnc_add_css_file (void)
     if (var)
     {
         gchar *str;
-        str = g_build_filename (var, ".gtk-3.0-gnucash.css", (char *)NULL);
+        str = g_build_filename (var, ".config/.gtk-3.0-gnucash.css", (char *)NULL);
```

Comment:
MacOS is BSD, so g_get_user_config_dir works on MacOS as it does on Linux and vanilla BSD: If XDG_CONFIG_HOME (it seldom is on MacOS) is set it uses that otherwise it uses ~/.config.
Apple thinks the right place is actually ~/Library/Application Support/Gnucash and that we should be forcing all of the various config stuff--aqbanking, gtk, whatever--to go there.

Reply:
@jralls would it make sense to propose a patch to glib to have it return ~/Library/Application Support for g_get_user_config_dir and g_get_user_data_dir for OS X ? Unless overridden by XDG_abc_HOME environment variables of course. For the record I also think they made the wrong choice on Windows. glib prefers the local APPDATA_DIR csid instead of the roaming one. They were clearly not thinking their software would be used in an Active Directory managed network...